psl/psl/tests/config/datasources.rs

```diff
@@ -216,6 +216,59 @@ fn postgresql_extension_parsing() {
     assert_eq!(None, uuid_ossp.schema());
 }
 
+#[test]
+fn empty_schema_property_should_error() {
+    let schema = indoc! {r#"
+        generator js {
+          provider        = "prisma-client-js"
+          previewFeatures = ["multiSchema"]
+        }
+
+        datasource ds {
+          provider   = "postgres"
+          url        = env("DATABASE_URL")
+          schemas = []
+        }
+    "#};
+
+    let expect = expect![[r#"
+        [1;91merror[0m: [1mIf provided, the schemas array can not be empty.[0m
+          [1;94m-->[0m  [4mschema.prisma:9[0m
+        [1;94m   | [0m
+        [1;94m 8 | [0m  url        = env("DATABASE_URL")
+        [1;94m 9 | [0m  schemas = [1;91m[][0m
+        [1;94m   | [0m
+    "#]];
+
+    expect.assert_eq(&parse_unwrap_err(schema));
```

Comment:
I think this can be replaced by `expect_error(schema, &expect)`. This would also fit within the new validation test suite.

Reply:
Done